frontend/src/components/listeners/Listeners.tsx

```diff
@@ -100,14 +103,14 @@ export function Listeners(props: ListenerProps) {
 		setSelectedCategory("");
 
 		if (searchText !== "") searchListeners(searchText, 0, limit, 0);
-		else listListeners(0, limit, 0, selectedCategory, selectedLabel, aliveOnly);
+		else listListeners(0, limit, 0, selectedCategory, selectedLabel, aliveOnly, process);
```

Comment:
Some how my search on dataset still show file extractors
<img width="1763" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/1eef005a-101d-4ede-9858-0d90f3d3a21b">

Reply:
I'll take a look at the search endpoint when I'm back Friday

Reply:
I think this is fixed now, can you check again?